pkg/kvstore/store/store.go

```diff
@@ -459,7 +459,10 @@ func (s *SharedStore) watcher(listDone chan bool) {
 
 			case kvstore.EventTypeDelete:
 				if localKey := s.lookupLocalKey(keyName); localKey != nil {
-					logger.Warning("Received delete event for local key. Re-creating the key in the kvstore")
+					if !option.Config.DryMode {
```

Comment:
Why is this needed?

Reply:
There are a ton of these logs that appear during the unit tests - do you think this is something to look into as a potential bug vs. a dry mode problem? The tests still pass, so I figured that it was simply a noisy-log problem relative to how we run the unit tests.

Reply:
Removed for now.